README.md

```diff
@@ -1,7 +1,6 @@
 # personal-portfolio
 
-In this project i built a personal portfolio page mobile version. In this project i added icons, used the flex box and offset properties where i used to postion ellements and css grid to display the various cards.Also I created a contact form where users can input different informations about themselfs and submit to formspree
-account which i created.no complex code is done here.
+In this project i built a desktop portfolio version webpage with a min width of 768px. grid and flex position elements are used to position elements in my webpage. Also on buttons hover was applied to them .no complex code is done here.
 ## Built With
 
```

Comment:
Kindly, I recommend putting your name in the Readme where it says "Author1"

Reply:
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

Reply:
hi @franclobo thank you for your correction. please do you mean i should add my name to the READme where it says AUTHOR1?

Reply:
Hello, @Ngufor-emmanuella yes please, every suggestion from the last reviewer is required unless it has the prefix **[OPTIONAL]**. I have already finished my review, you can check it out now. 😃